### PR TITLE
Update Mentor model and ProgramState with requested changes

### LINKED ISSUE #59: Change the program state MENTEE_APPLY to MENTEE_APPLICATION
**Is your feature request related to a problem? Please describe.**
It'll be more readable if we can change the program state MENTEE_APPLY to MENTEE_APPLICATION

**Describe the solution you'd like**
Update the ENUM

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### LINKED ISSUE #60: Update scholarx database tables
**Is your feature request related to a problem? Please describe.**
Few changes are requested for the scholarX database

**Describe the solution you'd like**
Add a column for mentor application
Rename questionnaire to the prerequisite

**Describe alternatives you've considered**
N/A

**Additional context**
https://sef.discourse.group/t/design-the-initial-erd-of-scholarx/72

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

The purpose of this PR is to fix #60 
The purpose of this PR is to fix #59 

## Goals

Add a column for mentor application
Rename mentor  questionnaire to the prerequisite
Change the program state MENTEE_APPLY to MENTEE_APPLICATION

## Approach

Updated programState ENUM
Updated mentor model

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
 
N/A

## Test environment
 
Ubuntu 19.04
OpenJDK 1.8
Tomcat 7

## Learning

N/A

### CODE REVIEW

File: src/main/java/org/sefglobal/scholarx/model/Mentor.java [hunk 1]
```diff
@@ -15,20 +15,31 @@
...
+        this.application = application;
+    }
+
+    public String getPrerequisites() {
+        return prerequisites;
     }
 
-    public void setQuestionnaire(String questionnaire) {
-        this.questionnaire = questionnaire;
+    public void setPrerequisites(String questionnaire) {
```
Comment (YohanAvishke) [inline src/main/java/org/sefglobal/scholarx/model/Mentor.java:41]:
```suggestion
    public void setPrerequisites(String prerequisites) {
```

**Review by YohanAvishke — CHANGES_REQUESTED** [review]

File: src/main/java/org/sefglobal/scholarx/model/Mentor.java (hunk 1 above)
Comment (Piumal1999) [reply src/main/java/org/sefglobal/scholarx/model/Mentor.java]: Oh sorry. I'll fix it

Comment (Piumal1999) [reply src/main/java/org/sefglobal/scholarx/model/Mentor.java]: Oh sorry. I'll fix it

Comment (Piumal1999) [reply src/main/java/org/sefglobal/scholarx/model/Mentor.java]: Fixed

**Review by YohanAvishke — APPROVED** [review]
NIce job @Piumal1999 😍

**Pull request merged**